### PR TITLE
bump version to 2.7.0 and update release notes

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by wpoely86** [comment]
:+1:

File: RELEASE_NOTES [hunk 1]
```diff
@@ -3,9 +3,70 @@ For more detailed information, please see the git log.
...
...),  astropy (#2724, #2727), attr (#2706),  BamUtil (#2654),  BBMap (#2322),  BH (#2508),  CheMPS2 (#2445),  CosmoloPy (#2723, #2727), csvkit (#2639),  Firefox (#2648),  FreeXL (#2422),  GL2PS (#2667),  Glade (#2631),  htop (#2538),  IGV (#2019),  IGVTools (#2019),  ImageMagick (#2438),  jModelTest (#2529),  KEALib (#2420),  libcerf (#2656),  libgcrypt (#2201),  libglade (#2631),  libgpg-error (#2201),  libspatialite (#2431), LittleCMS (#2438),  MAST (#2542),  MLC (#2577),  MPJ-Express (#2529),  mutil (#2201),  neon (#758),  NextClip (#2544),  npstat (#2686, #2703),  Octopus (#2643),  QuickFF (#2721), p4vasp (#2328),  PCMSolver (#2445),  PFFT (#2643),  PHYLIP (#2694),  pkgconfig (#2475, #2476),  Platypus (#2618),  pplacer (#1056),  PRINSEQ (#2437, #2444, #2585),  PyFFmpeg (#2501, #2519),  PyGObject (#2443),  PyGTK (#2443),  PyOpenGL (#2628),  pyringe (#2533),  qrupdate (#2675),  rgeos (#2635),  rpmrebuild (#2402),  shift (#2201),  SNAPE-pooled (#2688),  SNPhylo (#2701),  sratoolkit (#2715),  STAR-Fusion (#2463),  statsmodels (#2719),  StringTie (#2527),  synchronicity (#2508),  testpath (#2461),  USEARCH (#2537),  VarScan (#2464),  vsc-install (#2165),  Whoosh (#2725), xprop (#2645)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: where are my CRLFs?!? ;-)
(I think they got converted into spaces somewhere in that pipeline! ...)

Comment (boegel) [inline RELEASE_NOTES]: ah, thanks, I forget about that part...

File: RELEASE_NOTES [hunk 2]
```diff
@@ -3,9 +3,80 @@ For more detailed information, please see the git log.
...
+    - add missing libxml2 dependencies in GLib easyconfigs (#2658)
+    - fix Xerces-C++ download location (#2668)
+    - enable XML::Bare in all Perl easyconfigs (#2672)
+    - update dead link for SuiteSparse (#2679)
+    - remove custom exts_filter in easyconfigs used PythonPackage easyblock (#2683, #2685)
+    - add M4 as build dep for binutils & flex (#2681)
+    - add missing dependencies in Python 3.5.x easyconfigs: SQLite, Tk, GMP (#2704)
+    - fix (OS) deps, add checksums, remove parameter definition with default values in MVAPICH2 easyconfigs (#2707)
+- style cleanup in various easyconfigs (#2378, #2387, #2395, #2396, #2488-#2493, #2496-#2500, #2502-#2504, #2602)
+    - working towards automated style review of pulll requests
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: "pull" . we also need an automated spell checker ;-)

**Comment by fgeorgatos** [comment]
lgtm btw

**Pull request merged**